web-registry/src/pages/MyEcocredits/MyEcocredits.tsx

```diff
@@ -102,206 +95,78 @@ export const MyEcocredits = (): JSX.Element => {
     error,
     setError,
   } = useMsgClient(handleTxQueued, handleTxDelivered, handleError);
+
+  const { api } = useLedger();
```

Comment:
I believe we could actually get rid of this, it's only used for checking `api?.msgClient` in the hooks, which is already handled in `useMsgClient`

Reply:
You're right I will remove it.